code/modules/mob/living/carbon/human/life.dm

```diff
@@ -820,6 +835,35 @@
 		bodytemperature += recovery_amt
 
 	//This proc returns a number made up of the flags for body parts which you are protected on. (such as HEAD, UPPER_TORSO, LOWER_TORSO, etc. See setup.dm for the full list)
+/mob/living/carbon/human/proc/add_hypothermia(var/amount)
+	if(hypothermia+amount<=0)

```

Comment:
for each of the operators below, +, <=, >, +=, etc, add spaces around them pls
so it looks like `if(hypothermia + amount <= 0)`

Reply:
aaaa